tests/run_tests.py

```diff
@@ -52,12 +52,89 @@
     # DXC not currently building on bot
     "draw_triangle_list_hlsl.amber"
    ]
-}
+ }
+
+SUPPRESSIONS_DAWN = [
+  "compute_accumulated_ubo_definition.amber",
```

Comment:
Can you add a comment as to why things are suppressed? Grouping them is fine and a single #not implemented is also fine. Just, if there are long lived suppressions knowing why is handy.